classes/class-wpcom-liveblog-entry-key-events.php

```diff
@@ -114,6 +115,14 @@ public static function add_key_event_json( $json, $entry ) {
 
 		$json['key_event'] = self::is_key_event($json['id']);
 
+		// Add key event URL if it exists
+		if ( $json['key_event'] ) {
+			$key_event_url = get_comment_meta( $json['id'], self::META_KEY_URL, true );
+			if ( ! empty( $key_event_url ) ) {
+				$json['key_event_url'] = $key_event_url;
```

Comment:
should this be sanitized here, or does the escaping happen elsewhere?

Reply:
I do `esc_url` prior to saving: https://github.com/hrkhal/liveblog/pull/18/files#diff-d261b87cfa9200cd48f1e766e6497379R225

But there should really be escaping here, as well. I'll add it.